hugegraph-mysql/src/main/java/com/baidu/hugegraph/backend/store/mysql/MysqlTable.java

```diff
@@ -117,7 +124,11 @@ protected void createTable(Session session, TableDefine tableDefine) {
     }
 
     protected String engine() {
-        return " ENGINE=InnoDB";
+        StringBuilder sql = new StringBuilder();
+        sql.append(" ENGINE=");
+        sql.append(this.sessions.config().get(MysqlOptions.STORAGE_ENGINE));
+        LOG.debug("MySQL storage engine innodb/rocksdb: {}", sql.toString());
```

Comment:
remove unused log or normalize the log format,
and it's ok to just return `" ENGINE=" + session.config().get(MysqlOptions.STORAGE_ENGINE)`